redisearch/client.go

```diff
@@ -148,6 +148,16 @@ type IndexingOptions struct {
 	Replace  bool
 }
 
+// MultiError Represents one or more errors
+type MultiError map[int]error
+
+func (e MultiError) Error() string {
```

Comment:
not a blocker, but wouldn't you in this case rather chain all error strings somehow?